root/static/scripts/common/MB/Control/EditSummary.js

```diff
@@ -20,15 +20,15 @@ MB.Control.EditSummary = function (container) {
 
   self.addNote = function () {
     $toggleEditNote
-      .text(lp('Remove Note', 'Edit note field toggle'))
+      .text(lp('Remove note', 'Edit note field toggle'))
```

Comment:
I didn’t find where this is displayed but the context seems to be superfluous if not wrong:
```suggestion
      .text(l('Remove note'))
```

Reply:
It's a toggle on the side of the edit listing that alternates between "Add note" and "Remove note" but it only toggles whether the field to write the note is there - so I wasn't sure if it should be different from an "Add note" that submits the note and a "Remove note" that removes an existing note?

Reply:
Thanks, I found this one, but I can find the other one, as the button to submit a note seems to be labeled with “Submit note”. Anyway these are just buttons, I don’t really see the need to differentiate this one.

Reply:
It feels a bit like a different context but I guess the translations are likely to be the same, so sure, I'll change to 'button/menu' (which we already use for "Add note" elsewhere).